core/src/test/java/io/questdb/test/griffin/engine/table/parquet/PartitionEncoderTest.java

```diff
@@ -86,7 +86,8 @@ public void testBadVersion() throws Exception {
                     PartitionEncoder.encodeWithOptions(partitionDescriptor, path, COMPRESSION_UNCOMPRESSED, false, 0, 0, 42);
                     Assert.fail();
                 } catch (Exception e) {
-                    Assert.assertTrue(Chars.contains(e.getMessage(), "Invalid value for Version"));
+                    System.err.println(e.getMessage());
+                    Assert.assertTrue(Chars.contains(e.getMessage(), "unsupported parquet version 42"));
```

Comment:
nit: there is more convenient `TestUtils#assertContains()` method which also print the original string on mismatch, so that the above `System.err.println()` is no longer needed.

Reply:
Done. Good tip.